module/Finna/src/Finna/RecordDriver/SolrCommonFinnaTrait.php

```diff
@@ -95,6 +95,19 @@ protected function sanitizeHTML(
         return $result;
     }
 
+    /**
+     * Returns the mapped copyright if the copyright has been mapped to another
+     * copyright in configuration, otherwise returns the original copyright.
+     *
+     * @param string $copyright Copyright
+     *
+     * @return string
+     */
+    public function getMappedRights($copyright)
+    {
+        return $this->mainConfig['RightsMap'][strtoupper($copyright)] ?? $copyright;
```

Comment:
Kun on pienikin mahdollisuus, että stringissä on muutakin kuin perus-asciita, niin `mb_strtoupper($copyright, 'UTF-8')`

Reply:
Pitäisikö nuo lukuisat samassa yhteydessä olevat `$this->getRightsLink(strtoupper($copyright), $language))` kutsutkin muuttaa samalla?

Reply:
Joo, onhan se fiksumpi niin, että tehdään yhdessä paikassa. Asetuksen kommentissa voisi myös mainita, että kaikki avaimet ovat isoilla.